controllers/managedocs_controller.go

```diff
@@ -1067,6 +1085,52 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) getPrometheusRemoteWriteSpec() (*promv1.RemoteWriteSpec, error) {
+	if err := r.get(r.rhobsSecret); err != nil {
+		return nil, fmt.Errorf("unable to fetch rhobs secret: %v", err)
+	}
```

Comment:
I think you need to handle NotFound differently then any other error

Reply:
DOne